lib/compress/zstd_lazy.c

```diff
@@ -804,9 +803,10 @@ U16 ZSTD_rotateRight_U16(U16 const value, U32 count) {
  * value to reflect the update. Essentially cycles backwards from [0, {entries per row})
```

Comment:
You should update this comment

Reply:
I thought I did, thank you for noticing!